src/style.css

```diff
@@ -0,0 +1,43 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url("images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
let's merge same styles for classes 

```suggestion
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2) {
  background-image: url("images/star-active.svg");
}
```

Reply:
fixed:)

Reply:
But you didn't add any commits....🥲

<img width="842" alt="image" src="https://user-images.githubusercontent.com/86183007/215501509-34d31549-a1ff-43b7-b0c0-ab3e8fb7a415.png">